storyruntime/processing/Lexicon.py

```diff
@@ -325,6 +327,59 @@ def unless_condition(logger, story, line):
             return line['exit']
         return line['enter']
 
+    @staticmethod
+    async def try_catch(logger, story, line):
+        next_line = story.next_block(line)
+
+        if next_line is None:
+            return None
+
+        try:
+            await Lexicon.execute_block(logger, story, line)
+        except BaseException as e:
```

Comment:
Only StoryscriptError exceptions should be subjected to Storyscript try/catch blocks. All other errors are a result of incorrect runtime behaviour.
```suggestion
        except StoryscriptError as e:
```